tests/e2e/rosa_autoscaler_test.go

```diff
@@ -471,7 +471,7 @@ var _ = Describe("Autoscaler", labels.Feature.Autoscaler, func() {
 							"--ignore-daemonsets-utilization",
 							"--max-node-provision-time", "10m",
 							"--balancing-ignored-labels", "aaa",
-							"--max-nodes-total", "1000",
+							"--max-nodes-total", "10",
```

Comment:
Can we set a higher value here? Once the autosccaler removing failed, it will cause other case failure